tests/keystore/test_keypairs.py

```diff
@@ -52,15 +55,15 @@ def test_keypair_serialization():
     assert pubkey_bytes == bytes(umbral_pubkey)
 
     pubkey_b64 = new_keypair.serialize_pubkey(as_b64=True)
-    assert pubkey_b64 == umbral_pubkey.to_bytes()
+    assert pubkey_b64 == base64.urlsafe_b64encode(umbral_pubkey.to_bytes())
 
 
 def test_keypair_fingerprint():
     umbral_pubkey = UmbralPrivateKey.gen_key().get_pubkey()
     new_keypair = keypairs.Keypair(public_key=umbral_pubkey)
 
     fingerprint = new_keypair.fingerprint()
-    assert fingerprint != None
+    assert fingerprint is not None
```

Comment:
This makes me a bit uncomfortable since `not None == True`.  `is not` is not `!=`

Reply:
What value can `fingerprint` have in order to have different results for these assertions?

Reply:
Oh, it's totally valid, Personally, my eyes just tend to read this as `fingerprint is (not None)`.  PEP8 agrees with you though :-) https://www.python.org/dev/peps/pep-0008/#programming-recommendations.